README.md

```diff
@@ -40,9 +40,40 @@
 2. Choose an accent from the flavor directory
 3. Copy the contents of the accent directory into `~/.config/mpv`.
 
+## uosc support
```

Comment:
I don't think we should clutter the main README by copying the template again, maybe put this into its own `uosc` sub-directory?